synapse/handlers/device.py

```diff
@@ -499,7 +499,7 @@ async def delete_devices(self, user_id: str, device_ids: List[str]) -> None:
             else:
                 raise
 
-        # Delete access tokens and e2e keys for each device. Not optimised as it is not
+        # Delete data specific to this device. Not optimised as it is not
```

Comment:
really nitty wording preference:

We're deleting multiple devices here, so `each` makes more sense.
```suggestion
        # Delete data specific to each device. Not optimised as it is not
```